Maya/Exporter/BabylonExporter.Mesh.cs

```diff
@@ -919,15 +919,24 @@ private GlobalVertex ExtractVertex(MFnMesh mFnMesh, int polygonId, int vertexInd
                 {
                     MVector tangent = new MVector();
                     mFnMesh.getFaceVertexTangent(polygonId, vertexIndexGlobal, tangent);
+                    tangent.normalize();
+                    
+                    if (tangent.length < 0.1)
```

Comment:
How can length be less than 0.1 after normalization? Should you check this before calling `normalize`?